src/status_im/ui/screens/views.cljs

```diff
@@ -113,9 +113,25 @@
                           :paste-json-text paste-json-text
                           :add-rpc-url add-rpc-url
                           :network-details network-details
-                          (throw (str "Unknown view: " current-view)))]
+                          (throw (str "Unknown view: " current-view)))
+              iphone-x?   (and ios?
+                               (let [{:keys [width height]} (-> (.get dimensions "window")
+                                                                (js->clj :keywordize-keys true))]
+                                 ;; iPhone X dimensions = 375 x 812
+                                 (and (= width 375) (= height 812))))]
           [(if android? menu-context view) common-styles/flex
-           [view common-styles/flex
+           [(if iphone-x? safe-area-view view) (merge common-styles/flex
+                                                      (when iphone-x?
+                                                        ;; iPhone X needs a background color to match the current view
+                                                        {:background-color
+                                                         (case current-view
+                                                           (:wallet
+                                                            :wallet-send-transaction
+                                                            :wallet-request-transaction
+                                                            :choose-recipient) common-styles/color-blue4
+                                                           (:accounts :login) common-styles/color-blue2
+                                                           :transparent)
+                                                         :margin-top -20}))
```

Comment:
Seems that the new `safe-area-view` RN component doesn't allow us to set the padding size for top and bottom, so I come up with this kinda odd `:margin-top -20` to do the trick.